testing/insulating_clients.rst

```diff
@@ -8,35 +8,56 @@ If you need to simulate an interaction between different clients (think of a
 chat for instance), create several clients::
 
     // ...
+    use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
     use Symfony\Component\HttpFoundation\Response;
 
-    $harry = static::createClient();
-    $sally = static::createClient();
+    class MyTest extends WebTestCase
+    {
+        // ...
+        public function testWithMultipleClients()
+        {
+            self::ensureKernelShutdown();
+            $harry = static::createClient();
 
-    $harry->request('POST', '/say/sally/Hello');
-    $sally->request('GET', '/messages');
+            self::ensureKernelShutdown();
+            $sally = static::createClient();
```

Comment:
What's the point of the second client? It's using the same parameters, same kernel etc as the one before. So you could just use the same client for both the following requests or not?

```
$harry = static::createClient();
self::ensureKernelShutdown();
$sally = static::createClient();
```
This also seems pretty strange. How does `$harry` client still work when the kernel is shutdown?

Reply:
I guess this "works" because https://github.com/symfony/symfony/blob/829566cdea90a0be0231a15a229df2166868fce0/src/Symfony/Bundle/FrameworkBundle/KernelBrowser.php#L155
boots the kernel again. So harry starts a client and boots the kernel, then shutdown the kernel, then boot it again...